### ISSUE TITLE
Plan migration of REST HAL functionality to Java

### ISSUE THREAD

**Issue by paulo-reichert** [GalateaRaj/fuse-starter-java#191]
Issue Description
--
Currently HAL REST functionality is implemented [here](https://docs.google.com/spreadsheets/d/1uPDhdrcD6pFP150XhA-DJ-GIRMF1jVXZ_VJHmppLag4/edit?usp=sharing). When Slack queries Google scripts using the /metWith or /hal bindings, Google reads the corresponding command in the spreadsheet and returns the response associated with that request text. 

All of HAL commands should be natural language queries processed using the processor to be implemented in #189. All responses should be queriable data either in the HAL database (as per #185) or external web services.

The current commands are:
- movie quotes
- show bench (no. of Galateans not currently in client projects)
- count of Galateans
- coin flip
- lessons learned
- recommended reading

Other issues should be created detailing how we will migrate each of these features to HAL 2.0.

Design
--
[Description of the root cause for the issue and how to resolve it. Should be complete before Review is approved]

Test Evidence
--
[include relevant tests and output. Should be complete before Review is approved]

Validation in Develop
-- 
[Proof issue is resolved in Develop. Should be complete before Issue is closed]

**Comment by mikegajda** [comment]
Changing the status to st:already-done to remove from st:ready backlog in preparation for furlough work on fuse. If this was incorrect, I will reopen.

Note: This migration seems to have been completed by HAL 2.0 - Sprint 1

**Comment by mikegajda** [comment]
Reopening pending final decision from Steering Committee

**Comment by mikegajda** [comment]
Need to add a new item to do the actual HAL migration

**Comment by cbaltera** [comment]
Leaving this open as it appears the goal of this issue was to create the issue(s) in which to do the migration, and they have not been created.

**Comment by cbaltera** [comment]
Reverted to st:decision-needed.  Not sure _what_ decision is needed, so will move to st:ready if it seems appropriate, despite lack of urgency.

**Comment by cbaltera** [comment]
Consensus that decision was whether or not to do this; last week's discussion said yes but low priority, so moving to st:ready.
